h/static/scripts/guest.coffee

```diff
@@ -119,9 +120,27 @@ class Annotator.Guest extends Annotator
       # Announce the new positions, so that the sidebar knows
       this.plugins.Bridge.sync([highlight.annotation])
 
+  # Utility function to remove the hash part from a URL
+  _removeHash: (url) ->
+    url = new URL url
```

Comment:
> Can you [...] include a polyfill?

How about [this one](https://github.com/inexorabletash/polyfill/blob/master/url.js), by @inexorabletash ? (Listed [here](https://github.com/Modernizr/Modernizr/wiki/HTML5-Cross-Browser-Polyfills).)